Linker error with stage core

### BUG DESCRIPTION
Linker error with core feature/stage

```
Indexing .pioenvs/sonoff/libFrameworkArduino.a
Linking .pioenvs/sonoff/firmware.elf
/home/gitpod/.platformio/packages/toolchain-xtensa/bin/../lib/gcc/xtensa-lx106-elf/4.8.2/../../../../xtensa-lx106-elf/bin/ld: .pioenvs/sonoff/firmware.elf section `.text1' will not fit in region `iram1_0_seg'
collect2: error: ld returned 1 exit status
*** [.pioenvs/sonoff/firmware.elf] Error 1
============================================= [ERROR] Took 124.28 seconds =============================================
```


### REQUESTED INFORMATION

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Sonoff-Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Sonoff-Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in issues (https://github.com/arendst/Sonoff-Tasmota/issues)
- [x] Searched the problem in the wiki (https://github.com/arendst/Sonoff-Tasmota/wiki/Troubleshooting)
- [x] Searched the problem in the forum (https://groups.google.com/d/forum/sonoffusers)
- [x] Searched the problem in the chat (https://discord.gg/Ks2Kzd4)
- [x] Device used (i.e. Sonoff Basic) : none, compiler/linker error
- [x] Tasmota binary firmware version number used : latest dev 6.6.0.1
- [X] Development IDE - Compiler / VSC with platformIO 
- [ ] Provide the output of command ``status 0`` :

### TO REPRODUCE
_Steps to reproduce the behavior:_

Compiling with options enabled in platformio.ini

```
env_default = sonoff

platform                  = ${core_stage.platform}
build_flags               = ${core_stage.build_flags}

```

### EXPECTED BEHAVIOUR
Successfull linking

See https://github.com/arendst/Sonoff-Tasmota/issues/6045#issuecomment-509298834 too

![image](https://user-images.githubusercontent.com/24528715/60870490-45396f80-a231-11e9-93a7-fd9784320818.png)

--- a/sonoff/my_user_config.h
+++ b/sonoff/my_user_config.h
@@ -446,7 +446,7 @@
 
 #define USE_TX20_WIND_SENSOR                     // Add support for La Crosse TX20 anemometer (+2k code)
 
-#define USE_RC_SWITCH                            // Add support for RF transceiver using library RcSwitch (+2k7 code, 460 iram)
+//#define USE_RC_SWITCH                            // Add support for RF transceiver using library RcSwitch (+2k7 code, 460 iram)
 
 //#define USE_RF_SENSOR                            // Add support for RF sensor receiver (434MHz or 868MHz) (+0k8 code)
 //  #define USE_THEO_V2                            // Add support for decoding Theo V2 sensors as documented on https://sidweb.nl using 434MHz RF sensor receiver (+1k4 code)

